data/brands/club/scout.json

```diff
@@ -77,6 +77,16 @@
         "club": "scout"
       }
     },
+    {
+      "displayName": "Royal Rangers",
+      "locationSet": {"include": ["001"]},
```

Comment:
I don't think this "royal" things is suitable for all over the world……maybe you can list some major countries in locationSet

Reply:
I have found an [overview](https://rri.world/Member_nations.php) of the countries in which the Royal Ragers are active. I think there are 76 countries. Should I add them all to the locationSet?